core/net/packetbuf.h

```diff
@@ -236,6 +236,9 @@ enum {
   PACKETBUF_ATTR_MAX_MAC_TRANSMISSIONS,
   PACKETBUF_ATTR_MAC_SEQNO,
   PACKETBUF_ATTR_MAC_ACK,
+  PACKETBUF_ATTR_MAC_METADATA,
```

Comment:
I thought a name of a packetbuf attribute is not supposed to be specific to a particular protocol. That's way I named it `PACKETBUF_ATTR_METADATA` . I agree, this name alone may not give hints to readers what this attribute is for. I'll put a comment.